Tasks/Common/docker-common-v2/package-lock.json

```diff
@@ -9,7 +9,7 @@
       "resolved": "https://registry.npmjs.org/array-union/-/array-union-1.0.2.tgz",
```

Comment:
Undo these changes

Reply:
This was done automatically. Do we not want these changed?

Reply:
Yes.. The automatic changes generally don't fix the versions in the package-lock files.

Reply:
I'll undo this file, but can you help me understand. Why do we want the package-lock files that are checked in to differ from what we get with npm install?